src/main/java/services/BrowserValuesProvider.java

```diff
@@ -0,0 +1,22 @@
+package services;
+
+import beans.AuthenticatedRequestBean;
+import org.javarosa.core.model.utils.TimezoneProvider;
+
+/**
+ * Created by amstone326 on 1/8/18.
+ */
+public class BrowserValuesProvider extends TimezoneProvider {
```

Comment:
The reason I gave this a more generic name is because once this PR is merged, I think it makes sense to refactor the work I did for sending the browser location to formplayer to use this same pattern, rather than passing it down the chain in a million places.